src/components/gmServiceGmail.js

```diff
@@ -557,46 +557,49 @@ gmServiceGmail.prototype = {
           }
           
           try {
-            //var tbMatches = data.match(/\["tb",(?:.|\s)+?](?:\s]){2,}(?!\s,,)/g);   see: https://github.com/gmail-manager-community/gmail-manager-community/issues/4
-            var tbMatches = [];
+            var loc1 = data.indexOf("var VIEW_DATA=[[");
+            var loc2 = data.lastIndexOf("var GM_TIMING_END_CHUNK2");
+            var viewData = data.substring(loc1+14, loc2-2);
```

Comment:
@ath0mas: 
for debug, the data is basically gmail webpage's "view source" -- there may be some slight difference, its a very long string(about 260K here) .